README.md

```diff
@@ -45,11 +45,15 @@ Install-Package Umbraco.AuthorizedServices
 dotnet add package Umbraco.AuthorizedServices
 ```
 
-
 ### App Creation
 
 Services that this package are intended to support will offer an OAuth authentication and authorization flow against an "app" that the developer will need to create with the service.  From this various information will be available, including for example a "client ID" and "client secret" that will need to be applied in configuration.
 
+When creating the app it will usually be necessary to configure a call back URL. You should use the following:
+
+- For OAuth2: `/api/AuthorizedServiceResponse/HandleOAuth2IdentityResponse`
```

Comment:
I think this needs to be `/umbraco/api/AuthorizedServiceResponse/HandleOAuth2IdentityResponse` and `/umbraco/api/AuthorizedServiceResponse/HandleOAuth1IdentityResponse`

Reply:
Agreed - please can you change that and then merge if you are happy with the other changes/have confirmed I didn't break anything.

Reply:
Sure thing, I will merge them in after I run some tests with Flickr. Thank you.